.vitepress/config.js

```diff
@@ -290,6 +291,18 @@ export default defineConfig({
       ],
       
       "/script/": [
+     //  COMMENT: OPTIONAL 'SECTIONING' FORMAT OF INTRODUCTORY PAGES
```

Comment:
The idea of breaking those pages into sections is definitely something I'm interested in for the future. You can remove the commented out lines for now, but as we add more pages I do think it's somewhere we want to go.